coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -209,27 +208,6 @@ func (server *Server) AcquireJob(ctx context.Context, _ *proto.Empty) (*proto.Ac
 			}
 		}
 
-		// Compute parameters for the workspace to consume.
-		parameters, err := parameter.Compute(ctx, server.Database, parameter.ComputeScope{
-			TemplateImportJobID: templateVersion.JobID,
-			TemplateID: uuid.NullUUID{
-				UUID:  template.ID,
-				Valid: true,
-			},
-			WorkspaceID: uuid.NullUUID{
-				UUID:  workspace.ID,
-				Valid: true,
-			},
-		}, nil)
-		if err != nil {
-			return nil, failJob(fmt.Sprintf("compute parameters: %s", err))
-		}
-
-		// Convert types to their corresponding protobuf types.
-		protoParameters, err := convertComputedParameterValues(parameters)
-		if err != nil {
-			return nil, failJob(fmt.Sprintf("convert computed parameters to protobuf: %s", err))
-		}
```

Comment:
Maybe I'm just confused, but I thought we were still going to support Template-scoped parameters, just not workspace-scoped parameters?

Or, if we are getting rid of parameters entirely, then why not drop the database tables and/or the database queries related to them?

Reply:
> Maybe I'm just confused, but I thought we were still going to support Template-scoped parameters, just not workspace-scoped parameters?

Template-scoped parameters will be served differently, using [managed Terraform variables](https://coder.com/docs/v2/latest/templates/parameters#managed-terraform-variables).

> Or, if we are getting rid of parameters entirely, then why not drop the database tables and/or the database queries related to them?

1. Obviously, I like this idea but I left the data table on purpose if any customer faces a problem and would like to roll back to the previous release.
2. I use it to detect if the template still uses parameter variables.

Reply:
> We're going to drop any database queries that are no longer needed (anything that creates or updates legacy parameter data, read queries other than parameter schemas).

Done